setup/system.sh

```diff
@@ -144,6 +144,10 @@ pollinate  -q -r
 
 # Between these two, we really ought to be all set.
 
+echo 'Launching SSH public key creation...'
+
+nohup ssh-keygen -t rsa -b 2048 -a 100 -f /root/.ssh/id_rsa_miab -N '' -q 2>&1 >/dev/null
```

Comment:
The nohup command displays: 

nohup: ignoring input and redirecting stderr to stdout

The rest of the setup doesn't display irrelevant messages. Can this be hidden? Should it be nohup? We wait for the DH generation.